contracts/tether/Ante_USDTPegTest.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.0;
+
+import "../interfaces/IERC20.sol";
+import "../AnteTest.sol";
+import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+
+// Ante Test to check USDT remains > 0.90
+contract AnteUSDTPegTest is AnteTest("USDT is above 90 cents on the dollar") {
+    address public constant TetherAddr = 0xdAC17F958D2ee523a2206206994597C13D831ec7;
```

Comment:
Let's add a comment here that links to the etherscan token address for Tether.
https://etherscan.io/token/0xdAC17F958D2ee523a2206206994597C13D831ec7

Reply:
Link added!